src/pynwb/file.py

```diff
@@ -146,6 +149,31 @@ def __init__(self, **kwargs):
         for attr in recommended:
             setattr(self, attr, kwargs.get(attr, None))
 
+        io_to_wrap = []
+        for name, obj in getmembers(backends):
+            # find the backend modules
+            if ismodule(obj) & hasattr(obj, '__path__'):
```

Comment:
`&` -> `and`

Reply:
Could you add a `is_backend` function ? This seems to be a fragile way of checking if an object is  a backend